src/car/car.service.ts

```diff
@@ -28,4 +33,91 @@ export class CarService {
     }
     return avaliableCars;
   }
+
+  async checkAvgCarEmployment(dto: { id: number; month: string }) {
+    const oneDay = 24 * 60 * 60 * 1000;
```

Comment:
такие константы лучше писать в upper case 
```suggestion
    const ONE_DAY_IN_MS = 24 * 60 * 60 * 1000;
```